src/librustc/sir.rs

```diff
@@ -101,21 +102,20 @@ impl SirCx {
     pub fn funcs_and_blocks_deterministic(&self)
         -> IndexVec<SirFuncIdx, IndexVec<SirBlockIdx, *const BasicBlock>>
     {
-        // Scaffold a data structure which can be iterated deterministically by the caller.
-        // To start, all pointer addresses are None. We would have use overwritable null pointers,
-        // but Rust won't let us and the pointee is not sized.
+        // We start with a data structure where all LLVM block pointers are unknown (None).
         let mut res = IndexVec::from_elem_n(IndexVec::default(), self.funcs.len());
         for (func_idx, func) in self.funcs.iter_enumerated() {
             res[func_idx] = IndexVec::from_elem_n(None, func.blocks.len());
         }
 
-        // Now populate the pointers.
+        // Now we populate iterate over our hash table, replacing the aforementioned `None`s.
```

Comment:
"Now we populate iterate" doesn't parse?

Reply:
Fixed.